tests/quantizers_infrastructure_tests/keras_tests/test_inferable_quantizers/test_activations_inferable_quantizer.py

```diff
@@ -0,0 +1,358 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import unittest
+
+import numpy as np
+import tensorflow as tf
+
+from model_compression_toolkit import quantizers_infrastructure as qi
+from model_compression_toolkit.quantizers_infrastructure.common.constants import MULTIPLIER_N_BITS
+
+
+class TestKerasActivationsSymmetricQuantizer(unittest.TestCase):
+
+    def test_activation_signed_symmetric_inferable_quantizer(self):
```

Comment:
tests are very similar. better merge to one test and call with different quantizer types